runtime/doc/mlang.txt

```diff
@@ -37,22 +37,26 @@ use of "-" and "_".
 :lan[guage] mes[sages]
 :lan[guage] cty[pe]
 :lan[guage] tim[e]
+:lan[guage] col[late]
 			Print the current language (aka locale).
 			With the "messages" argument the language used for
 			messages is printed.  Technical: LC_MESSAGES.
 			With the "ctype" argument the language used for
 			character encoding is printed.  Technical: LC_CTYPE.
 			With the "time" argument the language used for
 			strftime() is printed.  Technical: LC_TIME.
+			With the "collate" argument the language used for
+                        collation order is printed.  Technical: LC_COLLATE.
```

Comment:
This line uses spaces for indentation whereas nearby line use tabs (inconsistent).

Reply:
thanks, fixed it as well.